presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestDistributedQueuesDb.java

```diff
@@ -57,7 +57,7 @@ public void setup()
         ImmutableMap.Builder<String, String> coordinatorProperties = new ImmutableMap.Builder<>();
         coordinatorProperties.put("query-manager.experimental.required-coordinators", "2");
         coordinatorProperties.put("resource-manager.query-heartbeat-interval", "10ms");
-        coordinatorProperties.put("resource-group-runtimeinfo-refresh-interval", "500ms");
```

Comment:
I tried with the timeout change but not with the refresh interval change in my local environment a few times, and I noticed that it could also pass the tests.
Are there any specific reasons that you want to change this parameter?